docs/user-quick-start.md

```diff
@@ -0,0 +1,299 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quick start guide for testing application DR
```

Comment:
Do we want to add "user" in the title to make it clear that this guide for users?